Bio/SeqIO/__init__.py

```diff
@@ -726,16 +721,10 @@ def read(handle, format, alphabet=None):
     to read multiple records from the handle.
     """
     iterator = parse(handle, format, alphabet)
-    try:
-        first = next(iterator)
-    except StopIteration:
-        first = None
+    first = next(iterator, None)
     if first is None:
         raise ValueError("No records found in handle")
-    try:
-        second = next(iterator)
-    except StopIteration:
-        second = None
+    second = next(iterator, None)
```

Comment:
This isn't quite as elegant as the pattern Michiel has been using in recent similar fixes, e.g. c0b9123f7f86205559d7a1a018a38d91898563fc